packages/schematics/angular/config/index.ts

```diff
@@ -17,9 +17,12 @@ import {
   strings,
   url,
 } from '@angular-devkit/schematics';
-import { AngularBuilder, readWorkspace, updateWorkspace } from '@schematics/angular/utility';
 import { posix as path } from 'node:path';
 import { relativePathToWorkspaceRoot } from '../utility/paths';
+// TODO: Determine if getWorkspace still needs to be imported as readWorkspace.
+import { getWorkspace as readWorkspace, updateWorkspace } from '../utility/workspace';
+// TODO: Determine if Builders still needs to be imported as AngularBuilder.
```

Comment:
out of curiosity: are these TODOs necessary or can we just drop them? seems fine to import that way

Reply:
I left them because I didn't want to make the decision if this is how they should be imported.  They are renamed on export to the public API.  They are unnecessary renames as far as I can tell in the inner workings of the library, but its not my area to know for sure.

Reply:
Right, but I worry nobody will act on these TODOs, so I would propose removing the TODOs and let review decide?